phoenix5-spark3/README.md

```diff
@@ -28,6 +28,18 @@ Apart from the shaded connector JAR, you also need to add the hbase mapredcp lib
 (add the exact paths as appropiate to your system)
 Both the `spark.driver.extraClassPath` and `spark.executor.extraClassPath` properties need to be set the above classpath. You may add them spark-defaults.conf, or specify them on the spark-shell or spark-submit command line.
 
+## Configuration properties
```

Comment:
Same comments apply as to the v2 readme,

Reply:
Done

Reply:
thanks